packages/calling-stateful-client/src/IncomingCallDeclarative.ts

```diff
@@ -24,12 +17,16 @@ export class ProxyIncomingCall implements ProxyHandler<DeclarativeIncomingCall>
   public get<P extends keyof IncomingCall>(target: IncomingCall, prop: P): any {
     switch (prop) {
       case 'accept': {
-        return this._context.withAsyncErrorTeedToState(async function (...args: Parameters<IncomingCall['accept']>) {
+        return this._context.withAsyncErrorTeedToState(async function (
+          ...args: Parameters<(IncomingCall | TeamsIncomingCall)['accept']>
```

Comment:
IncomingCall common, is agrs sharable? If not, ignore this comment

Reply:
IncomingCallCommon doesn't have any handlers on it, just the state types. each IncomingCallType has the same handlers with props but the proxy cannot see them if we use the common type here